Dockerfile

```diff
@@ -31,6 +32,7 @@ ENV E2E_FOCUS="Conformance"
 # configuration atm.  Fixes will be made upstream to resolve.
 ENV E2E_SKIP="Alpha|Disruptive|Feature|Flaky|Kubectl"
 ENV E2E_PROVIDER="local"
+ENV E2E_PARALLEL="n"
```

Comment:
how about a number instead?

Reply:
Can default to 1 instead, both have the same effect.